source/SIL.AppBuilder.Portal.Frontend/src/ui/components/inputs/product-definition-multi-select/index.tsx

```diff
@@ -1,13 +1,18 @@
-import { compose } from 'recompose';
+import { compose, withProps } from 'recompose';
 
 import { withTranslations } from '@lib/i18n';
 import { withNetwork as withProductDefinitions } from '@data/containers/resources/product-definition/list';
 import { withLoader } from '@data/containers/with-loader';
 
-import { Display } from './display';
+import { MultiSelect } from '@ui/components/inputs/multi-select';
 
 export default compose(
+  withTranslations,
   withProductDefinitions(),
   withLoader(({ error, productDefinitions }) => !error && !productDefinitions),
-  withTranslations,
-)(Display);
\ No newline at end of file
+  withProps(({ productDefinitions, t }) => ({
+    list: productDefinitions,
+    relationshipName: 'productDefinition',
```

Comment:
why is this needed?

Reply:
it's needed because I need to 
`const { data } = relationshipFor(selectedItem, relationshipName);`

Reply:
but what does it semantically represent